Python/pywarpx/callbacks.py

```diff
@@ -6,6 +6,7 @@
 #
 # License: BSD-3-Clause-LBNL
 
+# add this in the docs
```

Comment:
This string should show up in the docs. Not sure what exactly it currently is attached to, maybe the file it is in.

Reply:
yep, that's it